src/target/riscv/riscv.c

```diff
@@ -5093,6 +5098,38 @@ COMMAND_HANDLER(handle_reserve_trigger)
 	return ERROR_OK;
 }
 
+COMMAND_HANDLER(handle_riscv_mmu_translation_driver)
+{
+	struct riscv_info *info = riscv_info(get_current_target(CMD_CTX));
+	if (CMD_ARGC == 0) {
+		riscv_translation_driver_t driver = info->translation_driver;
+		command_print(CMD, "%s", riscv_translation_driver_to_str(driver));
+		return ERROR_OK;
+	}
+
+	if (CMD_ARGC != 1) {
+		command_print(CMD, "Command takes 0 or 1 parameters");
+		return ERROR_COMMAND_ARGUMENT_INVALID;
+	}
```

Comment:
Detail:

IIRC, the upstream reviewers prefer to return different error code in this case, and to not display any error message. That's because OpenOCD will show the command's help automatically when "SYNTAX_ERROR" is returned.

```suggestion
	if (CMD_ARGC != 1)
		return ERROR_COMMAND_SYNTAX_ERROR;
```

Reply:
Addressed